multi-server-adding-mailbox.adoc

```diff
@@ -0,0 +1,228 @@
+  directory on the new machine.
+* You are adding {product-short} Proxy to {product-abbrev}, this
+   should be installed on the existing single-server before you set up
+   the new mailbox server. See <<_installing_zimbra_proxy>>.
+
+
+== Overview of Process
+
+* {product-short} Mailbox Server is installed on the prepared machine.
+* Customized configuration for the single-server, such as custom themes
+  and Zimlets are added to the new mailbox server.
+* Commercial SSL certificates are added to the new mailbox server.
+* User accounts are moved from the single server to the new mailbox
+  server.
+* If you are moving all accounts from the single server, the mailbox
+  server is stopped on the single server machine.
+
+
+== Configuring the Mailbox Server
+
+The host name and zmhostname configured on the mailbox server is the
+same as on the name on the single server.
+
+Make sure you know the LDAP master password as you configure it on the
+sever that is being added. To find the master LDAP password on the
+single server node, type:
+
+----
+zmlocalconfig -s zimbra_ldap_password
+----
+
+IMPORTANT: Before you begin make sure you have an up-to-date backup!
+
+. Follow steps 1 through 4 in <<starting_multiple_server_install>>
+to log on to the server as root and unpack the
+{product-short} software.
+. Type `Y` for each package you are installing.
++
+--
+** Install `zimbra-store`, and `zimbra-spell` (optional) packages. When
+   `zimbra-spell` is installed, the `zimbra-apache` package also is
+   installed.
+** If `zimbra-proxy` is configured, install `memcached`.
+** The `zimbra-logger` package is installed only on one mailbox
+   server. If you are moving all mailboxes to this server from the
+   original single server, install the `zimbra-logger` package.
+** If _Archive and Discovery_ is installed on the single-server node,
+   install `zimbra-archiving` on the new mailbox server.
+
+NOTE: If SNMP is being used, type `Y` for `zimbra-snmp`. If SNMP is
+used, it is installed on every {product-short} server.
+--
+. Type `Y`, and press _Enter_ to modify the system. The selected packages
+are installed on the server.
++
+NOTE: The _Main_ menu displays the default entries for the {product-short}
+component you are installing.
+
+. Type `1` and press _Enter_ to go to the *Common Configuration* menu.
++
+NOTE: The mailbox server hostname is displayed. You must change the
+LDAP master host name and password to be the values configured on the
+single- server node.
++
+--
+** Type `2`, press _Enter_, and type the LDAP host name.
+** Type `4`, press _Enter_, and type the LDAP password.
+
+NOTE: After you set these values, the server immediately contacts the
+LDAP server. If it cannot contact the server, you cannot proceed.
+--
+
+** Type `6` to set the correct time zone.
+. Type `r` to return to the Main menu.
+. From the Main menu, type `2` to go to the Store configuration
+menu.
+** Type `4` and set the password for the administrator account. This
```

Comment:
I don't think this is correct. When you are installing a second mailstore node you should choose to NOT create admin user, because admin user has already been created when you installed the first mailstore. So, I think, that instead of "4 - set admin password", people should be choosing "2" to set "create admin user" to "no"